src/views/MarketsDropdown.tsx

```
@@ -497,12 +500,12 @@ const $Table = styled(Table)`
   }
 ` as typeof Table;
 
-const $Output = styled(Output)<{ isNegative?: boolean }>`
+const $Output = styled(Output) <{ isNegative?: boolean }>`
```

ESLint

[error] Delete `·`

(prettier/prettier)